webapp/components/Notification/Notification.vue

```diff
@@ -1,40 +1,90 @@
 <template>
-  <ds-space :class="{ read: notification.read, notification: true }" margin-bottom="x-small">
+  <ds-space :class="{ read: notificationData.read, notification: true }" margin-bottom="x-small">
     <client-only>
       <ds-space margin-bottom="x-small">
-        <user-teaser :user="from.author" :date-time="from.createdAt" />
+        <user-teaser :user="notificationData.triggerer" :date-time="notificationData.createdAt" />
       </ds-space>
       <ds-text class="reason-text-for-test" color="soft">
-        {{ $t(`notifications.reason.${notification.reason}`) }}
+        <base-icon
+          v-if="notificationData.report"
+          name="balance-scale"
+          v-tooltip="{ content: $t('notifications.report.name'), placement: 'right' }"
+        />
+        <base-icon
+          v-else-if="notificationData.comment"
+          name="comment"
+          v-tooltip="{ content: $t('notifications.comment'), placement: 'right' }"
+        />
+        <base-icon
+          v-else
+          name="bookmark"
+          v-tooltip="{ content: $t('notifications.post'), placement: 'right' }"
+        />
+        {{
+          $t(`notifications.reason.${notificationData.reason}` + notificationData.reasonExtention)
+        }}
       </ds-text>
     </client-only>
     <ds-space margin-bottom="x-small" />
     <nuxt-link
-      class="notification-mention-post"
-      :to="{ name: 'post-id-slug', params, ...hashParam }"
+      class="notification-link-for-test"
+      :to="notificationData.linkTo"
       @click.native="$emit('read')"
     >
       <ds-space margin-bottom="x-small">
         <ds-card
-          :header="from.title || from.post.title"
+          :header="(notificationData.post || notificationData.comment) && notificationData.title"
           hover
           space="x-small"
           class="notifications-card"
         >
-          <ds-space margin-bottom="x-small" />
-          <div>
-            <span v-if="isComment" class="comment-notification-header">
+          <div v-if="notificationData.user">
+            <!-- because of different margin above ds-card content without header property -->
+            <ds-space margin-bottom="small" />
```

Comment:
I guess we are trying to refactor out the `ds-space`... I think I could add `CSS` that would accomplish this, if you want me to, or @alina-beck ... we could just wait till @alina-beck has time to migrate it from the styleguide/refactor, but I don't mind helping out...

Reply:
Please go ahead or show me personally how you would do in CSS. @mattwr18